src/App.tsx

```diff
@@ -564,9 +564,6 @@ const Sidebar = (theme: Theme) => `
     background-size: 15px;
     background-position: center center;    
 
-    /*backdrop-filter: blur(40px);
-    -webkit-backdrop-filter: blur(40px);
-    background-color: rgba(256,256,256, 0.5) !important;*/
```

Comment:
Can get rid of this in favor of new `@supports` section it is left here intentionally?

Reply:
I've deleted this comment in some other PR I think :D but I can delete it here too.